<?xml version="1.0"?>
<launch>
    <param name="config" type="string" value="$(find lio_pkg)/config/velodyne_nclt.yaml" />
    <env name="LD_LIBRARY_PATH" value="$(find lio_pkg)/../../devel/.private/lio_pkg/lib:$(env LD_LIBRARY_PATH)" />
    <node pkg="lio_pkg" type="lio_iekf" name="lio_iekf" output="screen">
    </node>

    <!-- 启动 Rviz -->
    <node pkg="rviz" type="rviz" name="rviz" 
          args="-d $(find lio_pkg)/rviz/show.rviz" 
          output="screen"/>
</launch>